pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -425,9 +425,6 @@ def _clean_logger(trainer: "pl.Trainer") -> None:
         loggers = trainer.logger._logger_iterable if isinstance(trainer.logger, LoggerCollection) else [trainer.logger]
         for logger in loggers:
             if isinstance(logger, TensorBoardLogger) and logger._experiment is not None:
-                rank_zero_warn(
```

Comment:
I think if we are setting it to None below then we are creating a new experiment after log_hyperparams is called in the same trainer.fit call

Reply:
could this be a rank_zero_debug instead of a warning?

Reply:
The fix didn't work, we could remove it.